build/maven/jar-collector/pom.xml

```diff
@@ -16,7 +16,7 @@
   <parent>
     <groupId>org.geotools.maven</groupId>
     <artifactId>maven</artifactId>
-    <version>19-SNAPSHOT</version>
+    <version>18-SNAPSHOT</version>
```

Comment:
This doesn't look right for master. Why are you making this module depend on an earlier version of the parent?